jme3-core/src/main/resources/Common/MatDefs/Light/Deferred.j3md

```diff
@@ -61,4 +61,4 @@ MaterialDef Phong Lighting Deferred {
     Technique {
     }
 
-}
\ No newline at end of file
+}

```

Comment:
You can also restore this file, as it doesn’t contribute to the functionality of your PR